src/estimagic/optimization/tranquilo/sample_points.py

```diff
@@ -100,28 +95,29 @@ def _box_sampler(
     outside of the sampler.
 
     Args:
-        trustregion (TrustRegion): NamedTuple with attributes center and radius.
+        trustregion (Region): Trustregion. See module region.py.
         n_points (int): how many new points to sample
         rng (numpy.random.Generator): Random number generator.
         existing_xs (np.ndarray or None): 2d numpy array in which each row is an
             x vector at which the criterion function has already been evaluated, that
             satisfies lower_bounds <= existing_xs <= upper_bounds.
-        bounds (Bounds or None): NamedTuple.
 
     """
     n_params = len(trustregion.center)
-    effective_bounds = _get_effective_bounds(trustregion, bounds=bounds)
 
     points = rng.uniform(
-        low=effective_bounds.lower,
-        high=effective_bounds.upper,
+        low=trustregion.cube_bounds.lower,
```

Comment:
should probably evaluate `cube_bounds` once and then use its attributes to save repeated calculation.